index.html

```diff
@@ -257,6 +257,32 @@ <h1 class="second-text">About Myself</h1>
     </div>
 </div>
 </div>
+</section>
+
+<section class="img-b">
+  <img src="../image/header_background-v.png" alt="">
```

Comment:
You didn't add an alt text. The "alt" text adds a text description to an image on a web page.